lib/faraday_middleware/circuit_breaker/version.rb

```diff
@@ -1,5 +1,5 @@
 module FaradayMiddleware
   module CircuitBreaker
-    VERSION = "0.4.1"
+    VERSION = "0.4.2"
```

Comment:
Can you leave the version change and the changelog out of the PR? I usually do them on separate PRs :)